### DISCUSSION TITLE
@supports selector(:)

### DISCUSSION THREAD

**Discussion by Kallenju** [stylus/stylus#2715] [Q&A] [answered]
Hi,

I am new to Stylus. I use it only one day.

I'm trying to use `@supports selector(:focus-visible)`, however it doesn't works and results in a parser error. CSS @-rules like `@supports (display: flex)` work great.

Are there ways to use `@supports selector(:focus-visible)` in Stylus files?

**Comment by iChenLei** [comment, marked as answer]
Hi, Here is a workaround. https://stylus-docs.netlify.app/docs/literal.html . 
```styl
@css {
  @supports selector(:focus-visible) {
     /** You code here **/
   }
}
```
> I know this is a bug, we should fix it.

**Reply by Kallenju** [reply]
Thanks!
